apps/condo/domains/ticket/utils/serverSchema/exportTicketBlanksToPdf.js

```diff
@@ -256,15 +259,15 @@ const generatePdf = (replaces) => {
     const addLines = (count, startPosition) => {
         if (!count || !isNumber(count)) return null
 
-        return new Array(count).fill(1).map((_, index) => `<rect x="0" y="${startPosition + 14 + 15 * index}" width="400" height="1" fill="#707695"/>`)
+        return new Array(count).fill(1).map((_, index) => `<rect x="0" y="${startPosition + 14 + 15 * index}" width="${COMMON_PDF_SIZES.printWidth}" height="1" fill="#707695"/>`)
```

Comment:
mb move color to const?

Reply:
in task [DOMA-5104](https://jira.in.doma.ai/browse/DOMA-5104)